packages/bpk-component-drawer/src/BpkDrawerContent.tsx

```diff
@@ -34,42 +34,43 @@ import STYLES from './BpkDrawerContent.module.scss';
 const getClassName = cssModules(STYLES);
 
 type Props = {
-  children: Node,
-  dialogRef: () => mixed,
-  onCloseAnimationComplete: () => mixed,
-  onClose: () => mixed,
+  children: ReactNode,
+  dialogRef: () => React.RefObject<HTMLElement>,
+  onCloseAnimationComplete: () => void,
+  onClose: () => () => void
```

Comment:
Does it mean that this function returns a function which returns void?

Reply:
This is a typo - good spot. Fix pushed. 
(It is just in the type def so not affecting the close behaviour and the tests are either in TS and using mocks and tolerant to it or not migrated from JS yet so didn't highlight it.)